performance_manager/lib/gtfs_utils.py

```diff
@@ -18,13 +20,19 @@ def add_event_hash_column(
         "start_date",
         "start_time",
         "vehicle_id",
-    )
-    row_check = set(expected_hash_columns) - set(df_to_hash.columns)
+    ]
+
+    if isinstance(expected_hash_columns, list):
+        column_to_hash = expected_hash_columns
+    else:
+        column_to_hash = default_hash_columns
```

Comment:
This seems to make mypy happy, but I'm also open to other ideas.

Reply:
this is weird.

can `default_hash_columns` be initialized by default in the function signature from a constant list?

Reply:
mypy and pylint seem to be ok with this:
```
def add_event_hash_column(
    df_to_hash: pandas.DataFrame,
    hash_column_name: str = "hash",
    expected_hash_columns: Sequence[str] = (
        "is_moving",
        "stop_sequence",
        "stop_id",
        "direction_id",
        "route_id",
        "start_date",
        "start_time",
        "vehicle_id",
    ),
) -> pandas.DataFrame:
```